part-luks-1.sh

```diff
@@ -17,11 +17,9 @@
 #
 # Red Hat Author(s): Vendula Poncova <vponcova@redhat.com>
 
-# On RHEL the test is manual because of broken reading of the results from image.
-# Check the results on the running VM.
 # Ignore unused variable parsed out by tooling scripts as test tags metadata
 # shellcheck disable=SC2034
-TESTTYPE="skip-on-rhel storage partition luks"
```

Comment:
part-luks-* are failing for me for rhel, (I've just run them locally on rhel9 and all failed with RESULT not found). That is why I enabled them only on Fedora. Are they stable for you? I wonder if it can be just my environment (even though the tests are run in container)?

Reply:
Ah, I've just realized that my kstest runner container is 17 months (!) old. Let me try with something more recent :)

Reply:
Indeed, it works with current container!

Reply:
So let's enable the tests and see. If they show up unstable again, we can disable them. I won't do any harm.